test/Microsoft.NET.Build.Tests/GivenThatWeWantToVerifyProjectReferenceCompat.cs

```diff
@@ -0,0 +1,213 @@
+        [InlineData("netstandard1.5", "netstandard1.2", true)]
+        [InlineData("netstandard1.5", "netstandard1.3", true)]
+        [InlineData("netstandard1.5", "netstandard1.4", true)]
+        [InlineData("netstandard1.5", "netstandard1.5", true)]
+
+        [InlineData("netstandard1.6", "netstandard1.0", true)]
+        [InlineData("netstandard1.6", "netstandard1.1", true)]
+        [InlineData("netstandard1.6", "netstandard1.2", true)]
+        [InlineData("netstandard1.6", "netstandard1.3", true)]
+        [InlineData("netstandard1.6", "netstandard1.4", true)]
+        [InlineData("netstandard1.6", "netstandard1.5", true)]
+        [InlineData("netstandard1.6", "netstandard1.6", true)]
+
+        [InlineData("netstandard2.0", "netstandard1.0", true)]
+        [InlineData("netstandard2.0", "netstandard1.1", true)]
+        [InlineData("netstandard2.0", "netstandard1.2", true)]
+        [InlineData("netstandard2.0", "netstandard1.3", true)]
+        [InlineData("netstandard2.0", "netstandard1.4", true)]
+        [InlineData("netstandard2.0", "netstandard1.5", true)]
+        [InlineData("netstandard2.0", "netstandard1.6", true)]
+        [InlineData("netstandard2.0", "netstandard2.0", true)]
+
+        [InlineData("netcoreapp1.0", "netstandard1.0", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.1", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.2", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.3", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.4", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.5", true)]
+        [InlineData("netcoreapp1.0", "netstandard1.6", true)]
+        [InlineData("netcoreapp1.0", "netcoreapp1.0", true)]
+
+        [InlineData("netcoreapp1.1", "netstandard1.0", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.1", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.2", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.3", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.4", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.5", true)]
+        [InlineData("netcoreapp1.1", "netstandard1.6", true)]
+        [InlineData("netcoreapp1.1", "netcoreapp1.0", true)]
+        [InlineData("netcoreapp1.1", "netcoreapp1.1", true)]
+
+        [InlineData("netcoreapp2.0", "netstandard1.0", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.1", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.2", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.3", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.4", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.5", true)]
+        [InlineData("netcoreapp2.0", "netstandard1.6", true)]
+        [InlineData("netcoreapp2.0", "netstandard2.0", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp1.0", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp1.1", true)]
+        [InlineData("netcoreapp2.0", "netcoreapp2.0", true)]
```

Comment:
I agree that we could be more efficient with these added unit tests; I am looking into combining similar references sets into a single mutli-targeted unit test.